WalletWasabi.Fluent/ViewModels/Wallets/Advanced/WalletStatsViewModel.cs

```diff
@@ -25,10 +28,14 @@ public partial class WalletStatsViewModel : RoutableViewModel
 	[AutoNotify] private int _generatedUsedKeyCount;
 	[AutoNotify] private int _largestExternalKeyGap;
 	[AutoNotify] private int _largestInternalKeyGap;
+	[AutoNotify] private int _totalTransactionCount;
+	[AutoNotify] private int _transactionCount;
```

Comment:
I would rename to this to something like `_nonCoinjoinCount`

Reply:
@yahiheb The ones painted in blue in the transaction history grid are the "grouped" coinjoin transactions. I did not include those in the count. I can change that

Reply:
All those are coinjoin transactions, so they should be included.

Reply:
@yahiheb fixed in latest commit.